sdk/maps/azure-maps-render/src/samples/java/com/azure/maps/render/samples/RenderSamples.java

```diff
@@ -0,0 +1,265 @@
+        builder.httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS));
+
+        // Builds the client
+        RenderClient client = builder.buildClient();
+        // END: com.azure.maps.render.sync.builder.key.instantiation
+
+        return client;
+    }
+
+    public RenderClient createRenderSyncClientUsingAzureADCredential() {
+        // BEGIN: com.azure.maps.render.sync.builder.ad.instantiation
+        // Authenticates using Azure AD building a default credential
+        // This will look for AZURE_CLIENT_ID, AZURE_TENANT_ID, and AZURE_CLIENT_SECRET env variables
+        DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+        // Creates a builder
+        RenderClientBuilder builder = new RenderClientBuilder();
+        builder.credential(tokenCredential);
+        builder.mapsClientId(System.getenv("MAPS_CLIENT_ID"));
+        builder.httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS));
+
+        // Builds a client
+        RenderClient client = builder.buildClient();
+        // END: com.azure.maps.render.sync.builder.ad.instantiation
+
+        return client;
+    }
+
+    public RenderAsyncClient createRenderSearchAsyncClientUsingAzureKeyCredential() {
+        // BEGIN: com.azure.maps.render.async.builder.key.instantiation
+        // Authenticates using subscription key
+        AzureKeyCredential keyCredential = new AzureKeyCredential(System.getenv("SUBSCRIPTION_KEY"));
+
+        // Creates a builder
+        RenderClientBuilder builder = new RenderClientBuilder();
+        builder.credential(keyCredential);
+        builder.httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS));
+
+        // Builds the client
+        RenderAsyncClient client = builder.buildAsyncClient();
+        // END: com.azure.maps.render.async.builder.key.instantiation
+
+        return client;
+    }
+
+    public RenderAsyncClient createRenderSearchAsyncClientUsingAzureADCredential() {
+        // BEGIN: com.azure.maps.render.async.builder.ad.instantiation
+        // Authenticates using Azure AD building a default credential
+        // This will look for AZURE_CLIENT_ID, AZURE_TENANT_ID, and AZURE_CLIENT_SECRET env variables
+        DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+        // Creates a builder
+        RenderClientBuilder builder = new RenderClientBuilder();
+        builder.credential(tokenCredential);
+        builder.mapsClientId(System.getenv("MAPS_CLIENT_ID"));
+        builder.httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS));
+
+        // Builds a client
+        RenderAsyncClient client = builder.buildAsyncClient();
+        // END: com.azure.maps.render.async.builder.ad.instantiation
+
+        return client;
+    }
+
+    public static void main(String[] args) throws IOException {
```

Comment:
This is sample is too big and does too many things. We should split this into multiple smaller samples that demonstrate one scenario. See [these samples](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/containerregistry/azure-containers-containerregistry/src/samples/java/com/azure/containers/containerregistry).